src/auro-input.js

```diff
@@ -19,12 +27,13 @@ export default class AuroInput extends BaseInput {
      * @private Boolean value to determine which password icon to show
      */
     this.showPassword = false;
+    this.cardIcon = undefined;
```

Comment:
Please add `@private` to this as well. This does not need to be in the public API.

Reply:
Done